src/app/frontend/replicasetdetail/replicasetdetail.scss

```diff
@@ -37,13 +43,21 @@ $replicasetdetails-warning: #ffa500;
 }
 
 .kd-replicasetdetail-sidebar-header-icon {
+  color: $muted;
   font-size: 1.7em;
```

Comment:
Can you use $subhead-font-size-base? Or any other from https://github.com/angular/material/blob/8ef798f0fe242c555ef7c11fa0abe762fb218a02/src/core/style/variables.scss